hack/chglog/config.yml

```diff
@@ -2,15 +2,15 @@ style: github
 template: CHANGELOG.tpl.md
 info:
   title: CHANGELOG
-  repository_url: https://github.com/talos-systems/talos
+  repository_url: https://github.com/talos-systems/sidero
```

Comment:
not for this PR, but I would love us to switch to `release-tool` like Talos

Reply:
oh yeah, totally. I had some trouble with the release script this time. will add that as a 0.3 task.